test/integration/parachain/staging/fee.test.ts

```diff
@@ -33,33 +41,40 @@ describe("fee", () => {
         assert.equal(replaceGriefingCollateralRate.toString(), "0.1");
     }).timeout(2000000);
 
-    it("should getGriefingCollateral for issue", async () => {
-        const exchangeRateValue = new Big("3855.23187");
-        const exchangeRate = new ExchangeRate<
-            Bitcoin,
-            BitcoinUnit,
-            typeof collateralCurrency,
-            typeof collateralCurrency.units
-        >(Bitcoin, collateralCurrency, exchangeRateValue);
-        await callWithExchangeRate(oracleInterBtcAPI.oracle, exchangeRate, async () => {
-            const amountBtc = newMonetaryAmount(0.001, wrappedCurrency, true);
-            const griefingCollateral = await oracleInterBtcAPI.fee.getGriefingCollateral(amountBtc, GriefingCollateralType.Issue);
-            assert.equal(griefingCollateral.toBig(griefingCollateral.currency.base).round(5, 0).toString(), "0.00001");
-        });
-    }).timeout(200000);
-
-    it("should getGriefingCollateral for replace", async () => {
-        const exchangeRateValue = new Big("3855.23187");
-        const exchangeRate = new ExchangeRate<
-            Bitcoin,
-            BitcoinUnit,
-            typeof collateralCurrency,
-            typeof collateralCurrency.units
+    // TODO: re-enable and use mocking (see getPremiumRedeemVaults test for example)
+    it.skip("should getGriefingCollateral for issue", async () => {
```

Comment:
I think we could probably remove these tests but open to opinion. The most important thing for me is that the parsing and currency conversions are correct which could even be done in unit tests (if not already) instead of testing the setter and getter (which is provided by polkadot-js).

Reply:
Good shout, thank you. I'll add that note to the task for addressing this test.